parsec/core/fs/workspacefs/file_transactions.py

```diff
@@ -275,17 +260,15 @@ def _send_event(self, event, **kwargs):
             if source != (destination,):
                 await self._write_chunk(new_chunk, data)
 
-            # Update structures
-            removed_ids |= cleanup
-            result_dict[block] = new_chunk
+            # Craft and set new manifest
```

Comment:
C'est normal qu'on soit dans le `for` ici alors qu'on n'y était pas avant ?

J'imagine que c'est pour que chaque opération soit indépendante mais ça vaudrait peut être un commentaire si c'est important de faire de cette façon

Reply:
C'est pas capital, mais je me suis dis que c'était mieux de traiter le reshape de chaque bloc indépendamment. Par exemple, une annulation qui tomberait au milieu d'un reshape de manifest n'invalide que le reshape du bloc en cours et pas tout ceux d'avant.